src/main/java/org/fundacionjala/coding/denis/Ean.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.denis;
+
+public class Ean {
+    private int[] digitos;
+
+    public boolean kata() {
+        return digitos.length == 13 ? isEan() : false;
+    }
+
+    public boolean isEan() {
+        int sum = 0;
+        for (int i = 1; i < digitos.length; i++) {
+            sum += i % 2 == 0 ? digitos[i - 1] * 3 : digitos[i - 1] * 1;
+        }
+        System.out.println(sum);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)